roles/update-access/test/authorized_keys

```diff
@@ -0,0 +1,2 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDURd2gSSwnr41hcF3tlUSx17FJQHBB9a2pFj7qnDkSEYO6r7EVBBUihSXLovwSUqLNz08lJ+4ymd4eM/m9sKkMb0UgsGuTgxpRooIgawcmPOzCi5x8FLjAeI1jmlOpvKdqXK/3bILQUXSFsWXIklso9olBQ0VOPv+RuYVUcPzEhYjSIx8pEg4tmqWU6zDNt4LkYrH7CMwifkWw+Dh6fo+0J7kOotwFxCzqttboVYii9iEazSBNnm6RFSj/6MLsOKdYrqFLiJoVCJK0XSZo+oHHKjucK8G1ZkAOx97g7WDszz9XLj3GSKis+CJpjXnBtgBijV12oYWXs1/jlzCON8hT user1@test.lab
```

Comment:
I presume these are the keys in the .pub files in this same directory? Can you please add a comment per each line to indicate this?